app/assets/stylesheets/application.css

```diff
@@ -26,4 +26,5 @@
  *= require piecharts
  *= require @manageiq/react-ui-components/dist/textual_summary
  *= require @manageiq/react-ui-components/dist/tagging.css
+ *= require codemirror/theme/eclipse.css
```

Comment:
ping me once this is out of wip please

(working on moving codemirror off asset pipeline, so depending on how fast you are, this may have to change :))

Reply:
Actually, you can just remove this line.
It's already on line 13.